samples/tfm/tfm_hello_world/src/main.c

```diff
@@ -30,6 +35,25 @@ void main(void)
 	if (tfm_status != TFM_SUCCESS) {
 		printk("tfm_ns_interface_init failed with status %d\n", tfm_status);
 	}
+
```

Comment:
Idea: Since this is an instructional sample, maybe add a printout about what you are doing here. Could you add something for the crypto call as well :)?

Reply:
Added some prints on what is done.

```
	printk("Reading secure memory (FICR->INFO.PART)\n");
...
	printk("Reading secure memory (FICR->INFO.VARIANT)\n");
...
	printk("Hashing '%s'\n", hello_string);
```